db/migrate/20150224142024_rename_question_group_to_survey_in_rapidfire_questions.rb

```diff
@@ -0,0 +1,5 @@
+class RenameQuestionGroupToSurveyInRapidfireQuestions < ActiveRecord::Migration
+  def change
+    rename_column :rapidfire_questions, :question_group_id, :survey_id
+  end
+end
```

Comment:
okie, thats a fair point. so what about existing users who ran bunch of migrations already? i think they require these migrations.